Judge/src/main/java/pl/jakubowskiprzemyslaw/tajgertim/service/JudgeQueueListener.java

```diff
@@ -0,0 +1,38 @@
+package pl.jakubowskiprzemyslaw.tajgertim.service;
+
+import org.springframework.amqp.rabbit.annotation.RabbitListener;
+import org.springframework.stereotype.Service;
+import pl.jakubowskiprzemyslaw.tajgertim.models.round.NextRoundStatus;
+import pl.jakubowskiprzemyslaw.tajgertim.models.round.RoundStatus;
+import pl.jakubowskiprzemyslaw.tajgertim.models.shoot.PlayerShootResult;
+import pl.jakubowskiprzemyslaw.tajgertim.models.shoot.ShootResult;
+import pl.jakubowskiprzemyslaw.tajgertim.queues.Queues;
+import pl.jakubowskiprzemyslaw.tajgertim.services.LoggerService;
+import pl.jakubowskiprzemyslaw.tajgertim.services.QueueService;
+
+@Service
```

Comment:
Do you perhaps want to create your own annotation based on `@Component` that would be a listener? Listener doesn't strike me as a service TBH. Especially since it needs a service (QueueService AND LoggerService) to... well... perform a service. ;-)

Reply:
Done